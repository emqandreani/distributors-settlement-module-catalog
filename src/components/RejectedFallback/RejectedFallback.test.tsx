import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

import { RejectedFallback, RejectedFallbackProps } from "./";

const defaultProps: RejectedFallbackProps = {};

describe("< RejectedFallback />", () => {
  it("should render default without crash", () => {
    const tree = render(<RejectedFallback {...defaultProps} />);

    expect(tree).toMatchSnapshot();
  });
});
