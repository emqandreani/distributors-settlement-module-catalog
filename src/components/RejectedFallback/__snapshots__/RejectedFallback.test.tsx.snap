// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< RejectedFallback /> should render default without crash 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="root"
      >
        <h1>
          Por favor, reinicia tu sesión de usuario
        </h1>
        <a
          href="/login"
        >
          <button
            class="index_module_root__d6ce02ea MuiButton-root index_module_contained__d6ce02ea MuiButton-contained MuiButton-containedPrimary index_module_sizeMedium__d6ce02ea MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButtonBase-root  css-zddlty-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
            >
              Volver a login
            </h6>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="root"
    >
      <h1>
        Por favor, reinicia tu sesión de usuario
      </h1>
      <a
        href="/login"
      >
        <button
          class="index_module_root__d6ce02ea MuiButton-root index_module_contained__d6ce02ea MuiButton-contained MuiButton-containedPrimary index_module_sizeMedium__d6ce02ea MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButtonBase-root  css-zddlty-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          <h6
            class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
          >
            Volver a login
          </h6>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
